src/Components/test/testassets/BasicTestApp/GracefulTermination.razor

```diff
@@ -0,0 +1,13 @@
+﻿@inject NavigationManager navigationManager
+
+<h1>Graceful Termination</h1>
+
+<a href="mailto:test@example.com" id="mailto-link">Send Email</a>
+<button @onclick="DownloadFile" id="download-link">Download File</button>
+
```

Comment:
are there any other interesting href types we care about, javascript? `<a href="javascript:alert('Hello World!');">Execute JavaScript</a>`

Reply:
Good question! I don't think inline JavaScript is problematic here. Another scenario we should test is `hrefs` with `download` attributes. I'll push a commit with this soon.

Reply:
This test has been added.